selfdrive/loggerd/bootlog.cc

```diff
@@ -24,20 +25,22 @@ static kj::Array<capnp::word> build_boot_log() {
     i++;
   }
 
-  // Gather output of commands
-  std::vector<std::string> bootlog_commands = {
-    "[ -e /dev/nvme0 ] && sudo nvme smart-log --output-format=json /dev/nvme0",
-    "[ -x \"$(command -v journalctl)\" ] && journalctl",
-  };
+  if (!SIMULATION) {
```

Comment:
why is this hanging? I don't see why this is simulator specific; it doesn't hang on my machine

Reply:
I will do some more experimenting, if the issue persists, I will come back to this later.